packages/icons-workflow/bin/build.js

```diff
@@ -122,15 +124,16 @@ async function buildIcons(icons, tag, iconsNameList) {
             id = 'TextStrikethrough';
         }
 
-        let ComponentName = id === 'github' ? 'GitHub' : Case.pascal(id);
-
-        if (ComponentName === 'TextStrikeThrough') {
-            ComponentName = 'TextStrikethrough';
+        if (id === 'ChevronDownSize300') {
```

Comment:
Can we re-use the replacements map above, to avoid having to repeat this logic?

Reply:
done